subgraphs/aave-forks/src/sdk/manager.ts

```diff
@@ -728,6 +729,39 @@ export class DataManager {
     return liquidate;
   }
 
+  createTransferEvent(
```

Comment:
Will it work to just use the current `createTransfer` function in the `DataManager`? 

I wasn't expecting to have to update the sdk

Reply:
@dmelotik pushed the minimal changes that'd be needed in the sdk. Basically, skip creating account and position entities and not update usage if the sender is ZERO_ADDRESS (mint).

Now, shall I make these changes in the reference sdk, or consider this as a protocol specific diversion?